docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -26,6 +26,10 @@ If you are on Python 2, you'll first need to install a backport of the
 :ref:`running-unit-tests-dependencies` for details on installing `mock`_ and
 the other optional test dependencies.
 
+You also need to install pytz_ regardless of your Python version::
+
+   $ pip install pytz
```

Comment:
If Django adds more dependencies, this line will need to be modified each time as well.

Instead of re-specifying the dependencies here, we could consider suggesting an ["editable install"](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs). This would require that Django tests run in a venv, but that is good practice anyway.

For example, from the root of the project:

```bash
$ pip install -e .
```

This will use `setup.py` to install all Django dependencies.

This would also remove the need to specify `PYTHONPATH=..:$PYTHONPATH` in the `runtests.py` command below, as now Django would already be in the Python path.